Makefile

```diff
@@ -3,24 +3,28 @@ TEST?=$$(go list ./... | grep -v /vendor/ | grep -v teamcity)
 VETARGS?=-asmdecl -atomic -bool -buildtags -copylocks -methods -nilfunc -printf -rangeloops -shift -structtags -unsafeptr
 BUILD_TAGS?=${TOOL}
 GOFMT_FILES?=$$(find . -name '*.go' | grep -v vendor)
+GO_TEST_CMD?=go test -v
 
 # bin generates the releaseable binaries for this plugin
-bin: fmtcheck generate
+bin: fmtcheck
 	@CGO_ENABLED=0 BUILD_TAGS='$(BUILD_TAGS)' sh -c "'$(CURDIR)/scripts/build.sh'"
 
 default: dev
 
 # dev starts up `vault` from your $PATH, then builds the couchbase
 # plugin, registers it with vault and enables it.
 # A ./tmp dir is created for configs and binaries, and cleaned up on exit.
-dev: fmtcheck generate
+dev: fmtcheck
 	@CGO_ENABLED=0 BUILD_TAGS='$(BUILD_TAGS)' VAULT_DEV_BUILD=1 sh -c "'$(CURDIR)/scripts/build.sh'"
 
 # test runs the unit tests and vets the code
-test: fmtcheck generate
-	CGO_ENABLED=0 VAULT_TOKEN= VAULT_ACC= go test -v -tags='$(BUILD_TAGS)' $(TEST) $(TESTARGS) -count=1 -timeout=20m -parallel=4
+test: fmtcheck
+	CGO_ENABLED=0 VAULT_TOKEN= VAULT_ACC= ${GO_TEST_CMD} -tags='$(BUILD_TAGS)' $(TEST) $(TESTARGS) -count=1 -timeout=5m -parallel=4
```

Comment:
You can probably remove the VAULT_ACC variable, unless you're using it in your tests.